zephyr-apps/timer/src/main.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/drivers/gpio.h>
+#include <logging/log.h>
+#include <zephyr/kernel.h>
+LOG_MODULE_REGISTER(main);
+
+/* 1000 msec = 1 sec */
+#define SLEEP_TIME_MS   1000
+
+/* The devicetree node identifier for the "led0" alias. */
+// #define LED0_NODE DT_ALIAS(led0)
+#define SW0_NODE DT_ALIAS(sw0)
+
+/*
+ * A build error on this line means your board is unsupported.
+ * See the sample documentation for information on how to fix this.
+ */
+// static const struct gpio_dt_spec led = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
+static const struct gpio_dt_spec sw = GPIO_DT_SPEC_GET(SW0_NODE, gpios);
+
+void main(void)
+{
+	int ret;
+	int rising_edge = 1;
+	printk("timer started\n");
```

Comment:
Change `printk()` to using logging such as `LOG_INF()` if you add what I mentioned to the `prj.conf` logging should work for you

Reply:
will remove test printk once we get LOG_INF working